.github/workflows/daily_ci.yml

```diff
@@ -51,34 +51,28 @@ jobs:
       matrix:
         python-version: ["3.6", "3.7", "3.8", "3.9", "3.10", "3.11"]
     steps:
-      - name: Install pyenv and Python ${{ matrix.python-version }}
-        run: |
-          sudo apt-get install -y build-essential zlib1g-dev libffi-dev libssl-dev \
-            libbz2-dev libreadline-dev libsqlite3-dev liblzma-dev
-
-          curl https://pyenv.run | bash
-
-          export PYENV_ROOT="$HOME/.pyenv"
-          export PATH="$PYENV_ROOT/bin:$PATH"
-          eval "$(pyenv init -)"
-
-          pyenv install ${{ matrix.python-version }}
-
       - name: Checkout SiliconCompiler
         uses: actions/checkout@v3
         with:
           submodules: true
 
-      - name: Run tests with Python ${{ matrix.python-version }}
+      - name: Set tools and permissions for caching
         run: |
-          /start_slurm.sh
-          export PYENV_ROOT="$HOME/.pyenv"
-          export PATH="$PYENV_ROOT/bin:$PATH"
-          eval "$(pyenv init -)"
+          chown -R root: ~
+          ln -s $RUNNER_TOOL_CACHE /opt/hostedtoolcache
 
-          pyenv shell ${{ matrix.python-version }}
+      - name: Setup python ${{ matrix.python-version }}
+        env:
+          AGENT_TOOLSDIRECTORY: ${{ runner.tool_cache }}
+        uses: actions/setup-python@v4
+        with:
+          python-version: ${{ matrix.python-version }}
+          cache: 'pip'
 
-          python3 --version
+      - name: Run tests
+        run: |
+          /start_slurm.sh
+          
```

Comment:
could you add a call to `python3 --version` so we can ensure it's the correct version of python running?

Reply:
Sure added it back in now.